extension/src/toolbar/hooks/useLoadingModel.ts

```diff
@@ -0,0 +1,23 @@
+import { atom, useAtomValue, useSetAtom } from "jotai";
+import { useEffect, useMemo } from "react";
+
+import { readyAtom } from "../../prototypes/view/states/app";
+import { rootLayersLayersAtom } from "../../shared/states/rootLayer";
+
+const INITIAL_LAYER_LENGTH = 3;
+
+export const useLoading = () => {
```

Comment:
Sorry my document seems to be wrong... It's not working well. I wanted to show this loading modal until our data is ready, for example we need to wait for loading 3dtiles. But the loading modal is disappear soon. 
So could you change this logic like below? :bow:
1. You can remove this hooks
2. You can add `setReady` after [this line](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/blob/aa4fe23aa06df1fdae81c4d794fa174c4e10b97e/extension/src/shared/view/containers/InitialLayers.tsx#L71).
3. Then the loading modal should be appear until we load a initial data.